api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -838,3 +838,17 @@ message RequestIDExtension {
   // Request ID extension specific configuration.
   google.protobuf.Any typed_config = 1;
 }
+
+// Match input indicates that matching should be done on a specific request header.
+// TODO(snowp): Link to unified matching docs.
+message HttpRequestHeaderMatchInput {
```

Comment:
Does this handle multi-valued headers?  I think it should concatenate multiple values with ',' delimiters, and we should document that behavior.